<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shopdetail_sailer</class>
 <widget class="QWidget" name="shopdetail_sailer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="projectname_lb">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商品名</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>470</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="nodiscount_lb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 M</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>¥30</string>
    </property>
   </widget>
   <widget class="QLabel" name="discountold_lb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>¥30</string>
    </property>
   </widget>
   <widget class="QLabel" name="discountnew_lb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 M</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>¥10</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>470</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(76, 107, 34);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="discountprecent_lb">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>11</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 M</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-10%</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="icon_Fra">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>331</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="projectfather_lb">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>厂商名</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>651</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(82, 96, 111);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="buy_lb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 M</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>购买</string>
    </property>
   </widget>
   <widget class="QLabel" name="projectname_lb2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 M</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>商品名</string>
    </property>
   </widget>
   <widget class="QLabel" name="special_lb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>特价促销！</string>
    </property>
   </widget>
   <widget class="QLabel" name="still_lb">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 M</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>商品剩余：</string>
    </property>
   </widget>
   <widget class="QLabel" name="projectnumber_lb">
    <property name="geometry">
     <rect>
      <x>553</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 M</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="projectinfor_text">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 214, 214);</string>
   </property>
  </widget>
  <widget class="QLabel" name="projectfather_lb_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>店家：</string>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>projectname_lb</zorder>
  <zorder>frame_3</zorder>
  <zorder>frame_4</zorder>
  <zorder>icon_Fra</zorder>
  <zorder>projectfather_lb</zorder>
  <zorder>projectinfor_text</zorder>
  <zorder>projectfather_lb_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
